doc/release-notes.md

```diff
@@ -36,6 +36,13 @@ No attempt is made to prevent installing or running the software on Windows XP,
 you can still do so at your own risk, but do not expect it to work: do not
 report issues about Windows XP to the issue tracker.
 
+From 0.13.1 onwards OS X 10.7 is no longer supported. Issues with the libc++
```

Comment:
Should this be "from 0.13.0 onward"? My impression (from https://github.com/bitcoin/bitcoin/issues/8577#issuecomment-255945996 ) was that 0.13.0 already was more or less broken on 10.7?
@theuni